### PR TITLE
feat: add `PWA` support

### PR DESCRIPTION
**Pull request by hydrati**
I used the `Vite PWA` plugin to add PWA support to the site.
But to make the Service Worker cache update properly, you might need to configure `Cache Control`.

Preview:
Screenshot

### CODE REVIEW

**Comment by hydrati** [comment]
@sxzz it's ready. could you review this, thanks!

**Review by LinuxSuRen — COMMENTED** [review]
I'm not familiar with `WPA`. But we could remove the corresponding TODO item from [README.md](https://github.com/opensource-f2f/website#todo-list) once this PR could fix it.

**Comment by hydrati** [comment]
> I'm not familiar with `WPA`. But we could remove the corresponding TODO item from [README.md](https://github.com/opensource-f2f/website#todo-list) once this PR could fix it.

Thanks!

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+node_modules
+dist
```
Comment (sxzz) [inline .eslintignore:2]: They could be removed. Added here
https://github.com/sxzz/eslint-config/blob/8aa39cdbf54d54a67aadd3ece990e9cc2896e16b/packages/basic/index.js#L16-L32

**Review by sxzz — APPROVED** [review]

**Comment by sxzz** [comment]
Merged! thx a lot!

**Comment by LinuxSuRen** [comment]
Thanks! @hydrati

**Pull request merged**